File: backend-module/eneity/AICallLog.java
@Data
public class AICallLog {
    private int id;
    private int callerId;
    private String callerRole;
    private String functionName;
    private String aiModel;
    private String requestBody;
    private String responseBody;
    private int httpStatus;
    private int tokensInput;
    private int tokensOutput;
    private int tokensTotal;
    private double estimatedCost;
    private int durationMs;
    private int success;
    private String errorMessage;
    private String promptTemplate;
    private LocalDateTime createdAt;
    private boolean isDeleted;
}

File: frontend/src/types/aiCallLog.ts
export interface AiCallLog {
  id: number
  callerId: number
  callerRole: string | null
  functionName: string | null
  aiModel: string | null
  requestBody: string | null
  responseBody: string | null
  httpStatus: number | null
  tokensInput: number | null
  tokensOutput: number | null
  tokensTotal: number | null
  estimatedCost: number | null
  durationMs: number
  success: number
  errorMessage: string | null
  promptTemplate: string | null
  createdAt: string
  isDeleted: boolean
}
